### DISCUSSION TITLE
灌水：马上就要去原神FES，好紧张，但又有点兴奋

### DISCUSSION THREAD

**Discussion by 1451374815** [share121/inter-knot#1975] [其他]
马上就要去原神FES，好紧张，但又有点兴奋

**Comment by xjf0326** [comment]
你什么时候去
<img src="https://bbs-static.miyoushe.com/static/2023/05/23/a2fa805a0a782cdb8ab44ddaf8d5343d_962898009752203374.png" alt="阿姬倒地了04" width="75" >

**Comment by Genshin-Neuvillette** [comment]
慕了慕了
